README.md

```diff
@@ -1,50 +1,70 @@
-# Prism
+[![Prism is a set of packages for API mocking with OpenAPI v2 (formerly known as Swagger) and OpenAPI v3.](./examples/readme-header.svg)](https://stoplight.io/api-mocking?utm_source=github&utm_medium=prism&utm_campaign=readme)
 
 [![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/prism/master)](https://circleci.com/gh/stoplightio/prism)
 [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/prism-http?color=blue)](https://www.npmjs.com/package/@stoplight/prism-cli)
 [![Buy us a tree](https://img.shields.io/badge/Buy%20us%20a%20tree-%F0%9F%8C%B3-lightgreen)](https://offset.earth/stoplightinc)
 
-Prism is a set of packages for API mocking with **OpenAPI v2** (formerly known as Swagger) and **OpenAPI v3**.
-
-For example, given an API description document you can spin up a mock HTTP server and respond realistically based on your requests
+- **Mock Servers**: Life-like mock servers from any API Specification Document.
+- **Validation Proxy**: Contract Testing for API Consumers and Developers.
+- **Comprehensive API Specification Support**: OpenAPI 3.0, OpenAPI 2.0 (FKA Swagger) and Postman Collections support.
 
 ![Demo of Prism Mock Server being called with curl from the CLI](./examples/prism-cli.svg)
 
 > Note: This branch refers to Prism 3.x, which is the current version most likely you will use. If you're looking for the 2.x version, look at the [`2.x` branch][2.x]
 
-## Installation
+# Overview
+
+- [Installation and Usage](#-installation-and-Usage)
+- [Documentation and Community](#-documentation-and-community)
+- [Roadmap](#-roadmap)
+- [FAQs](#-faqs)
+- [Contributing](#-contributing) 
```

Comment:
I suggest to remove this, since keeping this up to date is something nobody will ever do (see platform internal) and links will point to the empty space.

A smart renderer should do this, not us directly in the document.

Reply:
<img width="300" alt="Screenshot 2020-12-22 at 12 41 35 PM" src="https://user-images.githubusercontent.com/17899454/102862848-8b998300-4453-11eb-97e6-c0b453bd39c1.png">

It looks like this in Stoplight which is weird. So to tackle this i created an overview section. That also makes navigating the readme easier on GitHub. Didn't add all the sections so there's not an urgent need to keep it up to date.